src/view/styles/ToolBar.css

```diff
@@ -142,20 +141,19 @@
     opacity: 90%;
     font-size: 14px;
     word-wrap: break-word;
+    padding-right: 10px;
```

Comment:
Just FYI, if you are worried about alignment you could also do:
```
text-align: justify;
text-justify: inter-word;
```
I am not sure if that's recommend from a UI standpoint, though.

Reply:
Ooo, thanks for the tip 😄